Mutagen.Bethesda.Analyzers.Api/IAnalyzer.cs

```diff
@@ -0,0 +1,206 @@
+﻿using System.IO.Abstractions;
+using Autofac;
+using Mutagen.Bethesda.Analyzers.Autofac;
+using Mutagen.Bethesda.Analyzers.Config.Run;
+using Mutagen.Bethesda.Analyzers.Config.Topic;
+using Mutagen.Bethesda.Analyzers.Drivers;
+using Mutagen.Bethesda.Analyzers.Reporting.Drops;
+using Mutagen.Bethesda.Analyzers.Reporting.Handlers;
+using Mutagen.Bethesda.Analyzers.SDK.Analyzers;
+using Mutagen.Bethesda.Analyzers.SDK.Drops;
+using Mutagen.Bethesda.Analyzers.Skyrim;
+using Mutagen.Bethesda.Environments;
+using Mutagen.Bethesda.Environments.DI;
+using Mutagen.Bethesda.Plugins;
+using Mutagen.Bethesda.Plugins.Meta;
+using Mutagen.Bethesda.Plugins.Records;
+using Noggog.WorkEngine;
+
+namespace Mutagen.Bethesda.Analyzers.Api;
+
+public static class AnalyzerExtensions
+{
+    public static IAnalyzer Analyzer(this IGameEnvironment gameEnvironment, AnalyzerOptions? options = null)
+    {
+        return new Analyzer(gameEnvironment, options);
+    }
+}
+
+public interface IAnalyzer
```

Comment:
I think we should rename this.  I view an analyzer as all the many analyzers you've been writing.

I think this component is more of an AnalyzerRunner?